mimic/model/clb_objects.py

```diff
@@ -490,6 +496,10 @@ def delete_nodes(self, lb_id, node_ids):
         if lb_id not in self.lbs:
             return not_found_response("loadbalancer"), 404
 
+        if len(node_ids) > self.batch_delete_limit:
```

Comment:
When you verified this, were you trying to delete 10 _valid_ node IDs or were you just deleting 10 random IDs?  My suspicion is that it validates the IDs before checking the length, but I just wanted to check :).